org.lflang/src/org/lflang/ASTUtils.xtend

```diff
@@ -162,6 +162,16 @@ class ASTUtils {
         return true
     }
     
+    /**
+     * Change the target name to 'newTargetName'.
+     * For example, change C to CCpp.
+     */
+    static def boolean changeTargetName(Resource resource, String newTargetName) {
```

Comment:
> I assume it is needed by the test that compile C tests as CCpp?

Correct.

> Maybe it would be better placed in the test package.

I think semantically, it belongs to ASTUtils because it is an AST transformation after all.
I think in terms of being dangerous, there will be validation and compiler errors if a target name is changed to an incompatible language, so it might be relatively safe perhaps.